docs/index.html

```diff
@@ -1022,6 +1022,31 @@ <h2 id="with-open-web-component-heading">With open web component</h2>
       </p>
     </div>
 
+    <div id="demo-global-trap-stack">
+      <h2 id="global-trap-stack-heading">Global trap stack</h2>
```

Comment:
Please add a short `<p>` after this heading to briefly explain what this example demonstrates.